Sources/TokamakLint/TokenVisitor.swift

```diff
@@ -0,0 +1,234 @@
+//
+//  TokenVisitor.swift
+//  SwiftSyntax
+//
+//  Created by Matvii Hodovaniuk on 4/2/19.
+//
+
+import Foundation
+import SwiftSyntax
+
+class TokenVisitor: SyntaxVisitor {
+  public var tree = [Node]()
+  public var current: Node!
```

Comment:
do these properties have to be public? also, do we need to use an implicitly unwrapped optional for `current`?